### PR TITLE
A5.5 http get call handling Education

### LINKED ISSUE #31: A5.5 HTTP Get Call Handling
As a User (Mentor, Mentee, or Educator), I want to be able to upload schools I attend(ed) and degrees or certificates I received to my profile

Details: Allow for the fetching of the User's education through GET API Call in order to display the name 

Technical Details: 
- Back-end
- MySQL
- Django

### PR DESCRIPTION
**Pull request by IanWearsHat**
Resolves #31 

Made GET call for getting all education rows in the entire table.
Made GET call for getting all education rows for a specific account.

### CODE REVIEW

**Review by Cynub — APPROVED** [review]
Looks good to me (no formatting issues, tests pass) 👍

File: mesh/education/urls.py [hunk 1]
```diff
@@ -3,6 +3,11 @@
 from . import views as education_views
 
 urlpatterns = [
-    # POST  /education/
-    path('', education_views.EducationView.as_view(), name="education")
+    # POST  /educations/
+    # GET   /educations/
+    path('', education_views.EducationView.as_view(), name="education"),
+
+    # GET   /educations/:account_id
```
Comment (JRSiwiecki) [inline mesh/education/urls.py:10]: Extremely small nitpick, but I believe I was just helping Brad with this issue, but clarify in this comment that the endpoint is /educations/:account_id/ with a trailing slash, as we were having problems where making requests to an endpoint that required a slash WITHOUT a slash, would not work correctly. So I would keep the trailing slash, just update the comment to match. Possibly a similar situation with the other two endpoints at /educations/

File: mesh/education/views.py [hunk 2]
```diff
@@ -14,17 +15,32 @@
...
+        Handle GET requests to /educations/ endpoint.
+
+        Returns a JSON response containing all Educations.
+        """
+        educations = Education.objects.all()
+
+        # Create a list of dictionaries containing education details
+        educations_list = serializers.serialize("json", educations)
+
+        return HttpResponse(educations_list, status=200)
```
Comment (JRSiwiecki) [inline mesh/education/views.py:29]: Good way to deal with the issue you were mentioning. 👍🏼

Comment (Cynub) [reply mesh/education/views.py:29]: Right? An HttpResponse instead of a JSON Response. +100000 points.

**Review by JRSiwiecki — APPROVED** [review]
Good work!

File: mesh/education/urls.py (hunk 1 above)
Comment (IanWearsHat) [reply mesh/education/urls.py:10]: I literally had this problem the first time I tried using Postman for this project. I will fix.

**Pull request merged**